.mailmap

```diff
@@ -199,6 +199,8 @@ Archit Verma <architv07@gmail.com>
 Arif Ahmed <arif.ahmed.5.10.1995@gmail.com>
 Arighna Chakrabarty <arighna.chakrabarty100@gmail.com>
 Arihant Parsoya <parsoyaarihant@gmail.com>
+Arnav Zutshi <84843295+AsRaNi1@users.noreply.github.com>
+Arnav Zutshi <arnzut1324@gmail.com>
```

Comment:
```suggestion
Arnav Zutshi <arnzut1324@gmail.com> Arnav Zutshi <84843295+AsRaNi1@users.noreply.github.com>
```
The current version will give two entries in the author list. The suggested change will map both emails to a single entry.